lib/rich-description.js

```diff
@@ -4,11 +4,17 @@ const renderer = new marked.Renderer()
 renderer.code = () => ''
 renderer.blockquote = () => ''
 renderer.heading = () => ''
-renderer.html = () => ''
+renderer.html = (src) => {
+  const match = src.match(/<br\/>/)
```

Comment:
should probably make it work with any `br` tag.

```suggestion
  const match = src.match(/<br\s*\/?>/)
```

Reply:
Thanks!